src/com/twu/biblioteca/MainMenu.java

```diff
@@ -45,26 +46,26 @@ public void showNoBooksMessage() {
 
 
     public void showOptions() {
-        for (int i = 0; i < options.size(); i++) {
-            System.out.println(options.get(i));
+        System.out.print(selectOption + "\n");
+        for (String option : options) {
```

Comment:
Even though we've not saved any lines of code here, not having to manage (i.e., initialise, increment, check) the `i` index makes this change worthwhile.